00-basics/002-javascript/Kristina/Learning002.js

```diff
@@ -0,0 +1,143 @@
+let RobotDinosaurFromSpace = {};
+RobotDinosaurFromSpace.roar = function(line) {
```

Comment:
Note that you can define fields inside original object definition already, e. g.:

```
let RobotDinosaurFromSpace = {
  roar: (line) => {
console.log(`The creature roars '${line}'`);
}
};
```